method: lattice_planner # lattice / em _planner
# method: em_planner

lattice_planner: {
  sample_max_time: 4.0,
  sample_min_time: 2.0,
  sample_time_step: 0.5,
  sample_lat_width: 4.75,      # 3.75
  sample_width_length: 0.25,

  weight_st_object: 10.0,    # 10.0
  weight_st_jerk: 5.0,       # 5.0
  weight_lt_offset: 100.0,   # 100.0
  weight_lt_acc: 5.0        # 5.0
}

em_planner: {
  #DP Path cost
  dp_sample_l: 0.5, # dp采样横向距离间隔
  dp_sample_s: 2.5, # dp采样纵向距离间隔
  dp_sample_rows: 9, # dp采样行数（横向）
  dp_sample_cols: 10,  # dp采样列数（纵向）

  dp_cost_collision: 1.0e+9, 
  dp_cost_dl: 50.0,    # 150.0
  dp_cost_ddl: 5.0,    # 10.0
  dp_cost_dddl: 1.0,    # 1.0
  dp_cost_ref: 10.0,    # 50.0

  #QP Path cost
  qp_cost_l: 15.0,       # 15.0
  qp_cost_dl: 5.0,     # 150.0
  qp_cost_ddl: 10.0,     # 10.0
  qp_cost_dddl: 1.0,     # 1.0
  qp_cost_ref: 10.0,     # 15.0
  qp_cost_end_l: 0.0,
  qp_cost_end_dl: 0.0,
  qp_cost_end_ddl: 0.0
  
#########速度规划相关############
#TO- DO
}